exercise/models.py

```diff
@@ -35,7 +35,19 @@ class ExerciseTask(PhishtrayBaseModel):
     score_threshold = models.IntegerField()
 
     def evaluate(self, score):
-        if score > self.score_threshold:
+        """
+        Constraints:
+            1. Scores should only range from 1 to 4.
+            2. Tasks should be unique in an exercise.
+            2. Tasks should not be shared between exercises.
+            3. Threshold is set percentage based with range 1-100.
+        :param score: INT / FLOAT
+            - this is calculated from a list of numbers with mean()
+        :return: STRING
+        """
+        scores = EmailReplyTaskScore.objects.filter(task=self)
```

Comment:
Is this used?

Reply:
yeah - https://github.com/cybsafe/phishtray/pull/165/files#diff-272fcb074afef6c25b809ae63a6eeda7R123

Reply:
good catch, it wasn't used